extensions/wikia/Discussions/maintenance/ForumDumper.php

```diff
@@ -356,6 +358,12 @@ public function getRevisions( $fh ) {
 						function ( $result ) use ( $dbh, $fh, &$inserts, &$firstRevsData ) {
 
 							while ($row = $result->fetchObject()) {
+
+								if ( $this->debug ) {
+									WikiaLogger::instance()->info( "Getting revision info for rev id: " .
+																   $row->rev_id . " in thread: " . $row->rev_page);
```

Comment:
if I may suggest something: put rev_id and rev_page into context fields instead of the message. It is much easier to query and filter kibana then

Reply:
Done, didn't know about it 👯